### PR TITLE
Room Tile DMs online/active green dot

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/riot-web/issues/4199

image

### CODE REVIEW

**Comment by t3chguy** [comment]
These will also be re-usable for the Member List of a room where the design shows an identical dot

File: src/components/views/rooms/RoomTileOnlineDot.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    useEventEmitter(user, "User.presence", currentlyActiveHandler);
+
+    return isOnline ? <span className="mx_RoomTile_online_dot" /> : null;
+};
+
+RoomTileOnlineDot.propTypes = {
+    userId: PropTypes.string.isRequired,
+};
+
+export default RoomTileOnlineDot;
```
Comment (turt2live) [inline src/components/views/rooms/RoomTileOnlineDot.js]: thank you for using the right exports - this makes merge conflicts less annoying.

**Comment by t3chguy** [comment]
image

File: src/components/views/rooms/UserOnlineDot.js [hunk 2]
```diff
@@ -38,11 +38,11 @@ const RoomTileOnlineDot = ({userId}) => {
     useEventEmitter(user, "User.currentlyActive", currentlyActiveHandler);
     useEventEmitter(user, "User.presence", currentlyActiveHandler);
 
-    return isOnline ? <span className="mx_RoomTile_online_dot" /> : null;
+    return isOnline ? <span className="mx_UserOnlineDot" /> : null;
 };
 
-RoomTileOnlineDot.propTypes = {
+UserOnlineDot.propTypes = {
```
Comment (turt2live) [inline src/components/views/rooms/UserOnlineDot.js]: imo unless we're using hooks or something that demands the use of a function-based component, we should be using classes. It's kinda important to be able to see the properties of the component at the top of the file.

Comment (t3chguy) [reply src/components/views/rooms/UserOnlineDot.js]: This *is* using hooks though

Comment (turt2live) [reply src/components/views/rooms/UserOnlineDot.js]: well I look dumb

**Pull request merged**